Assets/Scripts/Gameplay/Player/PlayerMovement.cs

```diff
@@ -56,7 +56,7 @@ void Update()
         }
         if (Input.GetKeyDown("h"))
         {
-            playerStats.CurrentHP = 0;
+            playerStats.TakeDamage(1020012);
```

Comment:
this should be wrapped in a way to control it doesn't enter a final build. YOu can use editor defines to help with this.